### PR TITLE
update several document related to privilege sql statement

### PR DESCRIPTION
**Pull request by imtbkcat**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by bb7133** [comment]
PTAL @yikeke

File: sql-statements/sql-statement-grant-privileges.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ aliases: ['/docs-cn/dev/reference/sql/statements/grant-privileges/']
 # GRANT <privileges>
 
 `GRANT <privileges>` 语句用于为 TiDB 中已存在的用户分配权限。TiDB 中的权限系统同 MySQL 一样，都基于数据库/表模式来分配凭据。
+执行 `GRANT <privileges>` 语句需要拥有需要分配的权限，并且拥有 `GRANT OPTION` 权限。
```
Comment (yikeke) [inline sql-statements/sql-statement-grant-privileges.md:11]:
```suggestion
执行 `GRANT <privileges>` 语句需要拥有分配的权限，并且拥有 `GRANT OPTION` 权限。
```

File: sql-statements/sql-statement-revoke-privileges.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ aliases: ['/docs-cn/dev/reference/sql/statements/revoke-privileges/']
 
 # REVOKE <privileges>
 
-`REVOKE <privileges>` 语句用于删除已有用户的权限。
+`REVOKE <privileges>` 语句用于删除已有用户的权限。执行 `REVOKE <privileges>` 语句需要拥有需要分配的权限，并且拥有 `GRANT OPTION` 权限。
```
Comment (yikeke) [inline sql-statements/sql-statement-revoke-privileges.md:10]:
```suggestion
`REVOKE <privileges>` 语句用于删除已有用户的权限。执行 `REVOKE <privileges>` 语句需要拥有分配的权限，并且拥有 `GRANT OPTION` 权限。
```

File: sql-statements/sql-statement-grant-privileges.md (hunk 1 above)
Comment (imtbkcat) [reply sql-statements/sql-statement-grant-privileges.md]: 对

**Comment by imtbkcat** [comment]
done @yikeke

**Pull request merged**